test/drone_controller/input_layer/test_drone_state.py

```diff
@@ -0,0 +1,62 @@
+from pytest import raises
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205178)